src/app.html

```diff
@@ -5,7 +5,7 @@
     <meta name="author" content="SbS e.V." />
     <meta
       name="description"
-      content="Ehrenamtliche Nachhilfe von Studenten für Schüler in deutschlandweit über 45 Unistädten."
+      content="Ehrenamtliche Nachhilfe von Studierenden für Schüler:innen in deutschlandweit über 50 Unistädten."
```

Comment:
You can overwrite this value in __layout.svelte if we fetched English microcopy using [`svelte:head`](https://svelte.dev/tutorial/svelte-head):

```svelte
<svelte:head>
	<meta
      name="description"
      content={$microcopy.description}
    />
</svelte:head>

Reply:
Included title, author, description and defer data-domain to be set with microcopy

Reply:
Excellent.